Jenkinsfile

```diff
@@ -73,6 +73,7 @@ pipeline {
 
     stages {
         stage('All') {
+
```

Comment:
are all of the changes in this file just linebreaks and formatting, or are there any value changes aswell?

Reply:
just one change with new gradle update the gradle task copyAndroidNatives is not working anymore

Reply:
ok, now i see it

Reply:
task copyAndrioidNatives is not being executed. so both paralles stages are executing the task.